dist/src/main/java/io/camunda/zeebe/restore/RestoreApp.java

```diff
@@ -21,11 +22,11 @@
 import org.springframework.boot.WebApplicationType;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.Import;
 
-@SpringBootApplication(
-    scanBasePackages = {"io.camunda.zeebe.restore", "io.camunda.zeebe.broker.shared"})
-@ConfigurationPropertiesScan(
-    basePackages = {"io.camunda.zeebe.broker.shared", "io.camunda.zeebe.restore"})
+@SpringBootApplication(scanBasePackages = {"io.camunda.zeebe.restore"})
+@ConfigurationPropertiesScan(basePackages = {"io.camunda.zeebe.restore"})
+@Import(value = {BrokerBasedConfiguration.class, WorkingDirectoryConfiguration.class})
```

Comment:
❓ The import allows us to remove the scan config?

Reply:
Yes, exactly. Previously, the `io.camunda.zeebe.broker` package had a `shared` package inside that was included as a base package in the `RestoreApp` so that Spring also includes these two configuration classes. I was messing around wit the `shared` package just to make the `RestoreApp` working, but eventually went this approach to keep it easy.

Coming back to your question, with the `@Import` annotation additional configuration classes can be included for the scanning.

Reply:
Thanks 👍🏼